docs/decisiones/gestor-tareas.md

```diff
@@ -0,0 +1,23 @@
+# Elección del gestor de tareas
+
+Para la elección del gestor de tareas tendré en cuenta:
+
+- que se use o se recomiende su uso para TypeScript o JavaScript.
+
+- que se mantenga actualizado regularmente a día de hoy.
+
+- que si puedo limitarme a usar el mínimo número de dependencias posible, mejor.
+
+- su dificultad en el aprendizaje.
+
+- su rendimiento
+
+Existen, entre otros muchos, los siguientes gestores de tareas:
+
+- __grunt__/__gulp__: gestores de tareas en los que se pueden automatizar tareas con el minimo esfuerzo en un archivo de configuración en formato JavaScript. Son bastante similares en cuanto a rendimiento. Hay que instalarlos como dependencia del proyecto.
+
+- __yarn__/__npm__/__bun__: son gestores de paquetes/dependencias pero también pueden usarse como gestores de tareas mediante la declaración de una clave "script" en el fichero `package.json`, por lo tanto es muy simple, fácil y rápido de usar. Permite paralelizar tareas con el fin de maximizar la utilización de recursos y permite una gran escalabilidad.
+
+- __make__: es un gestor de tareas con muy buen rendimiento en el que para declarar las tareas usa la sintaxis de bash, lo cual hace que su uso sea muy cómodo.
```

Comment:
Para declarar las tareas usa su propia sintaxis. Para *ejecutar* las tareas usa la sintaxis del shell que se le indique, [que no es `bash` sino `sh`](https://unix.stackexchange.com/questions/217243/which-shell-is-used-in-gnu-make-files).
Por favor, tratad de no poner cosas que no sean ciertas. Y menos cuando, como es el caso, no vienen a cuento. ¿Que use shell u otra cosa ayuda a elegirla? ¿Es un requisito de tu proyecto?

Reply:
> Para ejecutar las tareas usa la sintaxis del shell que se le indique, [que no es bash sino sh](https://unix.stackexchange.com/questions/217243/which-shell-is-used-in-gnu-make-files).

Es verdad, lo cambio en el siguiente commit.

> ¿Que use shell u otra cosa ayuda a elegirla? ¿Es un requisito de tu proyecto?

Es cierto, lo quito en el siguiente commit.